cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -330,7 +378,124 @@ public Response evaluateMeasure(@Context HttpServletRequest request,
 
 		return response;
 	}
-	
+
+	@POST
+	@Path("/evaluation-patient-list")
+	@Consumes(MediaType.MULTIPART_FORM_DATA)
+	@Produces({ MediaType.APPLICATION_JSON })
+	@ApiOperation(value = "Evaluates a measure bundle for a list of patient"
```

Comment:
Suggestion:   "Evaluates a measure bundle for a list of patients"  << patients plural.